app/controllers/veterans/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Veterans::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```

Comment:
Are the code comments included here useful? Are they a partial thrown away implementation?